home/index.html

```diff
@@ -8,83 +8,35 @@
 		body {
   			background-color: #aae9ff;
 		}
-		
-		#test {
-			float: left;
-			width: 250px; 
-			height: 538.5px;
-			padding-right: 10px;
-			}
-		#Ecode {
-			height: 100px;
-			width: 28%;
-			border: none;
-			border-radius: 2%;
-			resize: none;
-			float: left;
-			background: #aae9ff;
-			}
 		</style>
 	</head>
 	<body style="margin:0">
 		<div class="content">
-			<h1>Scratch Studio Embed</h1>
+			<h1>Scratch Studio Embed
+			<span id="DevTag">STABLE</span></h1>
```

Comment:
Logically, the easier accessible version is the stable one. You should either remove that, or start using the GitHub 's releases feature and number versionning (following the model MAJOR.MINOR.PATCH), and indicate the release number here.

Reply:
Fixed, I made it display "By -Accio-" instead of "STABLE"